### PR TITLE
cars-assemble concept exercise

### PR DESCRIPTION
**Pull request by colinleach**
A second exercise for the `conditionals` concept. This was written earlier, but `vehicle-purchase` may be a better first exercise on this concept.

### CODE REVIEW

File: exercises/concept/cars-assemble/.docs/introduction.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+In this case, parentheses `()` around the boolean and braces `{}` around the statement are both required. 
+A simple `if` statement can be shortened:
+
+```R
+if (x == 0) return("success")
+```
+
+An alternative if-else form may be useful:
```
Comment (jonmcalder) [inline exercises/concept/cars-assemble/.docs/introduction.md:37]: As before, I think it's better if `ifelse()` is omitted from the introduction.

File: exercises/concept/cars-assemble/.meta/design.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Goal
+
+The goal of this exercise is to teach the student about conditionals in R. It provides additional practice after `vehicle-purchase`.
+
+## Learning objectives
+
+- Know that the relational operators are `==`, `!=`, `>=`, `<=`.
+- Know the syntax of a block if-else.
+- Know about the `ifelse()` function.
```
Comment (jonmcalder) [inline exercises/concept/cars-assemble/.meta/design.md:11]: As before, remove `ifelse()` here.

**Review by jonmcalder — APPROVED** [review]
Please remove `ifelse()` in the 2 spots noted before merging.

File: exercises/concept/cars-assemble/.meta/design.md (hunk 2 above)
Comment (colinleach) [reply exercises/concept/cars-assemble/.meta/design.md:11]: I'll certainly make these changes. For practical reasons, probably tomorrow.

**Comment by colinleach** [comment]
`ifelse` has been removed from 2 files. I'll go ahead and merge.

**Pull request merged**